### PR TITLE
Sync criteria and deps title style

### LINKED ISSUE #1309: Sync criteria and deps title style

### PR DESCRIPTION
**Pull request by LamaEats**
Apply common styles for Criteria and Deps Tables and Forms

## PR includes

- [x] Bug Fix
- [x] Refactor

## Related issues

Resolve #1309 

## QA Instructions, Screenshots, Recordings
  
image

### CODE REVIEW

File: src/components/ActivityFeed.tsx [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import styled from 'styled-components';
 import { gapL, gapM, gray5 } from '@taskany/colors';
+import { nullable } from '@taskany/bricks';
```
Comment (awinogradov) [inline src/components/ActivityFeed.tsx:3]: image

File: src/components/GoalCriteria/GoalCriteria.tsx [hunk 2]
```diff
@@ -298,60 +208,70 @@ const GoalCriteriaItem: React.FC<GoalCriteriaItemProps> = memo((props) => {
...
+                                <Text size="s">{w}</Text>
+                            ))}
+                        </CustomCell>
+                    ),
+                },
+                { name: 'state' },
+                { name: 'projectId' },
+                {
+                    name: 'issuers',
+                    renderColumn: (values) => (
```
Comment (9teen90nine) [inline src/components/GoalCriteria/GoalCriteria.tsx:264]: `({ issuers })`

also I really hate render functions

File: src/components/GoalListItemCompact.tsx [hunk 3]
```diff
@@ -135,45 +165,51 @@ const Column: ColumnRender = ({ col, componentProps }) => {
...
+                        <UserGroup users={list} />
+                    ))}
+                </StyledCell>
             );
         case 'estimate':
             return (
-                <ContentItem {...columnProps}>
-                    <TextItem>{nullable(estimate, (e) => estimateToString(e))}</TextItem>
-                </ContentItem>
+                <StyledCell {...columnProps}>
```
Comment (9teen90nine) [inline src/components/GoalListItemCompact.tsx:208]: you can set a variable in this switch and render `<StyledCell>` once

File: src/components/GoalSuggest.tsx [hunk 4]
```diff
@@ -117,9 +118,9 @@ export const GoalSuggest = forwardRef<HTMLDivElement, GoalSuggestProps>(
                 value={value}
                 onChange={onSelectItem}
                 items={items}
-                visible={visible}
+                visible={visible && showSuggest}
                 renderInput={renderInput}
-                maxWidth={400}
+                maxWidth={600}
```
Comment (9teen90nine) [inline src/components/GoalSuggest.tsx:123]: maybe we should make this relative of some sort?

**Review by 9teen90nine — COMMENTED** [review]
Nothing critical, just some thoughts

File: src/components/GoalSuggest.tsx [hunk 5]
```diff
@@ -117,9 +118,9 @@ export const GoalSuggest = forwardRef<HTMLDivElement, GoalSuggestProps>(
                 value={value}
                 onChange={onSelectItem}
                 items={items}
-                visible={visible}
+                visible={visible && showSuggest}
```
Comment (awinogradov) [inline src/components/GoalSuggest.tsx:121]: What the practice to rename props? Can we not to do this? `visible` > `visible` — it's clear

File: src/components/GoalDependencyList/GoalDependencyList.i18n/ru.json [hunk 6]
```diff
@@ -1,5 +1,6 @@
 {
     "blocks": "Блокируют",
```
Comment (awinogradov) [inline src/components/GoalDependencyList/GoalDependencyList.i18n/ru.json:2]: Блокирует

File: src/components/GoalDependencyList/GoalDependencyList.tsx [hunk 7]
```diff
@@ -1,16 +1,17 @@
 import React, { useCallback, useMemo } from 'react';
 import styled from 'styled-components';
-import { CleanButton, MessageTextAltIcon, Text, nullable } from '@taskany/bricks';
-import { backgroundColor, gray6 } from '@taskany/colors';
+import { Text, nullable } from '@taskany/bricks';
+import { backgroundColor, danger0, gray4, gray6, gray9 } from '@taskany/colors';
```
Comment (awinogradov) [inline src/components/GoalDependencyList/GoalDependencyList.tsx:4]: `gray6` is not used

File: src/components/GoalSuggest.tsx (hunk 5 above)
Comment (LamaEats) [reply src/components/GoalSuggest.tsx:121]: `showSuggest` prop has changed after select any item fron suggest, this is not rename

File: src/components/GoalSuggest.tsx (hunk 4 above)
Comment (LamaEats) [reply src/components/GoalSuggest.tsx:123]: no, 'cause this property of Popup lib component, which calculating based on content width and value of prop

File: src/components/GoalCriteria/GoalCriteria.tsx (hunk 2 above)
Comment (LamaEats) [reply src/components/GoalCriteria/GoalCriteria.tsx]: well i could rewrite this with React.Context but eventually will get render function in consumer :)

**Review by awinogradov — APPROVED** [review]

**Review by 9teen90nine — APPROVED** [review]

**Pull request merged**